packages/apps-config/package.json

```
@@ -23,6 +23,7 @@
     "@sora-substrate/type-definitions": "^1.3.4",
     "@subsocial/types": "^0.5.2",
     "@zeitgeistpm/type-defs": "^0.1.51",
+    "thales-types-bundle": "0.8.2",
```

```suggestion
    "thales-types-bundle": "^0.8.2",
```

Just commit the `yarn.lock` as well. (I'm assuming that is why CI is failing, i.e. it doesn't auto-install, rather just keeps to the deps as added)